Python/Python.sublime-syntax

```diff
@@ -454,39 +455,33 @@ contexts:
       captures:
         1: punctuation.definition.numeric.binary.python
     # complex
```

Comment:
nit: Should this line say `imaginary` now?

Reply:
The comment? An imaginary is still a part of a complex number. Maybe someone is keen enough to add the real part in the future, too?

Reply:
Complex numbers in Python are constructed by adding any numeric with an imaginary number annotated with `j`, e.g. `1+2j`. There is a convention that suggests not to use spaces around this operator to indicate the creation of a complex number with a non-zero real part, which we may target, but that would incure duplicating basically every match pattern for numeric literals and I don't know what your intention with the scopes is.

Reply:
... that's why "keen" ... :-)  

Was discussed with @jwortmann in the Ruby PR and we agree it to make things too complex with only little or even no value.

Reply:
https://github.com/sublimehq/Packages/pull/2135#discussion_r337142160 for reference.

Reply:
My idea of how to scope complex numbers is at https://github.com/sublimehq/Packages/pull/2135#discussion_r338225323.

Reply:
> the key thing we want to highlight here is that the + is not an addition sign, but part of making the whole thing an imaginary number

This isn't super meaninful in python - constant folding means that `1+1j` is no more an atom for constructing complex numbers than `1 + 1 + 1j + 1j` is.

My original remark was simply that the comment should be updated to match the code change, maybe to "imaginary literals for complex numbers" or something.

Reply:
The ongoing discussion is just about some aspects of R or Ruby with regards of imaginary numbers being combined with other type modifiers like `r` for rational numbers or even hexadecimal expressions.